tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -278,9 +282,14 @@
 					<p>
 						<a id="jetpack-beta-branch-link" target="_blank" rel="nofollow noopener" href="#">…</a>
 					</p>
+					<p><h3>JurassicNinja on Atomic</h3></p>
```

Comment:
Personally I can't say I'm fond of the heading here, I'd leave it out like you did for the "trunk" case above. But I suppose it'll only be temporary, so :shrug:.